index.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Arrays are special variables that can hold multiple value at once.
+ * They are special type of object. Arrays are like containers that holds
+ * multiple items at once. We have different types of Array methods.
+ * 1. Push() 2. Pop() 3. unshift() 4. shift() 5. reverse() 6. sort()
+ * 7. indexOf 8. lastIndexOf() etc
+  */
+
+
+var colors = ["blue","green"]
+ //To append indigo to the end
+ colors[2] = ["indigo"]; /**or */ colors.push("indigo");
+console.log(colors);
+
+//replace the middle with red
+
+colors.splice(1,1,"red")
```

Comment:
this will replace only replace element in position 1 and not in the middle, kindly update this to match replacing array with old length element in the middle,